ui/page/privacy/account_mixer_page.go

```diff
@@ -68,175 +68,176 @@ func (pg *AccountMixerPage) OnNavigatedTo() {
+			mixedBalance = dcrutil.Amount(acct.TotalBalance).String()
+		} else if acct.Number == pg.wallet.UnmixedAccountNumber() {
+			unmixedBalance = dcrutil.Amount(acct.TotalBalance).String()
+		}
+	}
+	return l.Theme.Card().Layout(gtx, func(gtx C) D {
+		txt := l.Theme.H6(values.String(values.StrBalance))
+		return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
+			layout.Rigid(func(gtx C) D {
+				return layout.UniformInset(values.MarginPadding40).Layout(gtx, func(gtx C) D {
+					return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
+						layout.Rigid(func(gtx C) D {
+							return layout.Inset{Bottom: values.MarginPadding15}.Layout(gtx, func(gtx C) D {
+								return layout.Flex{Axis: layout.Horizontal, Alignment: layout.Middle}.Layout(gtx,
+									layout.Rigid(txt.Layout),
+									layout.Rigid(func(gtx C) D {
+										return layout.Inset{Left: values.MarginPadding5}.Layout(gtx, func(gtx C) D {
+											return components.LayoutBalance(gtx, pg.Load, pg.totalBalance.String())
+										})
+									}),
+									layout.Flexed(1, func(gtx C) D {
+										return layout.E.Layout(gtx, func(gtx C) D {
+											txt := l.Theme.H6(values.String(values.StrMix))
+											return layout.Flex{Alignment: layout.Middle}.Layout(gtx,
 												layout.Rigid(func(gtx C) D {
-													return components.LayoutBalance(gtx, pg.Load, mixedBalance)
+													return layout.Inset{Right: values.MarginPadding10}.Layout(gtx, func(gtx C) D {
+														return txt.Layout(gtx)
+													})
 												}),
+												layout.Rigid(button),
 											)
-										}),
-									)
+										})
+									}),
+								)
+							})
+						}),
+						layout.Rigid(func(gtx C) D {
+							return l.Theme.Separator().Layout(gtx)
+						}),
+						layout.Rigid(func(gtx C) D {
+							return layout.Inset{Top: values.MarginPadding15}.Layout(gtx, func(gtx C) D {
+								return layout.Flex{Axis: layout.Horizontal, Alignment: layout.Middle}.Layout(gtx,
+									layout.Rigid(func(gtx C) D {
+										txt := l.Theme.H6(values.String(values.StrMixer))
+										return txt.Layout(gtx)
+									}),
+									layout.Rigid(func(gtx C) D {
+										return layout.Inset{Left: values.MarginPadding20, Right: values.MarginPadding40}.Layout(gtx, func(gtx C) D {
+											return pg.mixerProgress.Layout(gtx)
+										})
+									}),
+								)
+							})
+						}),
+					)
+				})
+			}),
+			layout.Rigid(func(gtx C) D {
+				return layout.Inset{Right: values.MarginPadding15, Left: values.MarginPadding15, Top: values.MarginPadding15}.Layout(gtx, func(gtx C) D {
+					return layout.Flex{Axis: layout.Horizontal, Alignment: layout.Middle}.Layout(gtx,
+						layout.Rigid(func(gtx C) D {
+							ic := l.Theme.Icons.MixedTxIcon
+							return ic.Layout12dp(gtx)
+						}),
+						layout.Rigid(func(gtx C) D {
+							txt := l.Theme.H6(values.String(values.StrMixed))
+							return layout.Inset{Left: values.MarginPadding11}.Layout(gtx, func(gtx C) D {
```

Comment:
Should be `return layout.Inset{Left: values.MarginPadding11}.Layout(gtx, txt.Layout)`